validators/terravaloper1hqyygjq0vdqk0xdpkffgamzr4f7tqqg5u9mg66/README.md

```diff
@@ -0,0 +1,114 @@
+---
+Moniker: AngelProtocol
+Validator: terravaloper1hqyygjq0vdqk0xdpkffgamzr4f7tqqg5u9mg66
+Email: hi@angelprotocol.io
+Telegram: @AngelProtocol
+Twitter: @AngelProtocoI
```

Comment:
twitter handle wrong. Current @angelprotocoi, suggested change to @angelprotocol